### PR TITLE
Print error messsage if litani not installed

### PR DESCRIPTION
**Pull request by mianowill**
*Issue #, if available:* 
N/A

*Description of changes:*
Adds a helpful error message to the user if `litani` is not installed

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by karkhaz — APPROVED** [review]

File: src/cbmc_starter_kit/setup.py [hunk 1]
```diff
@@ -73,8 +73,12 @@ def main():
     source_root = repository.repository_root()
     if shutil.which("litani"):
         litani = Path("litani")
-    else:
+    elif repository.litani_root() is not None:
         litani = repository.litani_root() / "litani"
+    else:
+        print("Error: could not find litani root. Is litani installed?")
```
Comment (markrtuttle) [inline src/cbmc_starter_kit/setup.py:79]: I'm not sure if this should be a print or a `logging.error`.

**Review by markrtuttle — APPROVED** [review]

File: src/cbmc_starter_kit/setup.py (hunk 1 above)
Comment (mianowill) [reply src/cbmc_starter_kit/setup.py:79]: I think `logging.error` would make more sense actually, will add that in tomorrow.

**Comment by mianowill** [comment]
Thanks @karkhaz @markrtuttle for the reviews; could somebody approve running the CI workflows for this PR?

File: src/cbmc_starter_kit/setup.py [hunk 2]
```diff
@@ -73,8 +73,12 @@ def main():
     source_root = repository.repository_root()
     if shutil.which("litani"):
         litani = Path("litani")
-    else:
+    elif repository.litani_root() is not None:
         litani = repository.litani_root() / "litani"
+    else:
+        logging.error("could not find litani root. Is litani installed?")
```
Comment (markrtuttle) [inline src/cbmc_starter_kit/setup.py:79]: This is great.  Please consider an error message like "Can't find litani: see installation instructions at https://github.com/awslabs/aws-build-accumulator/releases/latest"

**Review by markrtuttle — CHANGES_REQUESTED** [review]

**Review by markrtuttle — APPROVED** [review]

**Pull request merged**